IPyth.sol

```diff
@@ -31,5 +31,15 @@ interface IPyth {
     /// @notice Update price feeds with given update messages if they are more recent than the current stored prices.
     /// The call will succeed even if the update is not the most recent.
     /// @dev Reverts if the updateData is invalid.
-    function updatePriceFeeds(bytes[] memory updateData) external;
+    /// @param updateData Array of price update data.
+    /// @param feeAmount Fee amount in Wei or Pyth token.
+    /// @param feeUsingPythToken True if fee is paid using Pyth Token and false if it's paid using the network's native currency.
+    function updatePriceFeeds(bytes[] memory updateData, uint feeAmount, bool feeUsingPythToken) external payable;
```

Comment:
I don't think we should have a bool here. Instead, we should be able to configure the IPyth contract deployment to set the token it expects as a fee.

Reply:
ok, actually, after looking at the whole thing, let's drop the ERC20 payment for now and we can add it back later as a separate function if we need it.

Reply:
+1